.github/workflows/UI tests.yml

```diff
@@ -0,0 +1,27 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
can you have the yml without space in the name of the file? like `ui_tests.yml` or `ui-tests.yml`, whichever you prefer as long as it has no spaces